bodhi/server/scripts/skopeo_lite.py

```diff
@@ -689,10 +689,16 @@ def _copy_blob(self, digest, size):
             self.src.download_blob(digest, size, self.dest.ensure_blob_path(digest))
         elif isinstance(self.src, DirectoryEndpoint) and isinstance(self.dest, RegistryEndpoint):
             self.dest.upload_blob(digest, size, self.src.get_blob_path(digest))
-        elif isinstance(self.src, RegistryEndpoint) and isinstance(self.dest, RegistryEndpoint):
-            if self.src.registry == self.dest.registry:
-                self.dest.link_blob(digest, self.src.repo)
+        else:
+            # The copy() function below ensures that the following two asserts don't fail.
+            assert (isinstance(self.src, RegistryEndpoint)
+                    and isinstance(self.dest, RegistryEndpoint))
+            assert self.src.registry == self.dest.registry
+
+            self.dest.link_blob(digest, self.src.repo)
+
         # Other forms of copying are not needed currently, and not implemented
+            raise RuntimeError(f"Can't copy between registries: {self.src!r}, {self.dest!r}")
```

Comment:
This line is currently breaking up the `bodhi/tests/server/scripts/test_skopeo_lite.test_skopeo_copy_link` test

Reply:
Oops, so much for "fixing" my changes :)